code/_onclick/click.dm

```diff
@@ -16,6 +16,28 @@
 	Note that this proc can be overridden, and is in the case of screen objects.
 */
 
+/client/Click(var/atom/target, location, control, params)
+//	if(world.time <= next_click) // Hard check, before anything else, to avoid crashing
+//		return
+
+//	next_click = world.time + 1
+
+//	if(buildmode && !istype(target, /obj/screen))
+//		buildmode.build_click(src.mob, params, target)
+//		return
+	if(!isHUDobj(target) && CH)
```

Comment:
Click handlers seem to be a new thing, did you port this from somewhere or write it yourself? Is it applicable to things other than changelings?

Reply:
What is the general purpose of it though? What else can it do? Why does a client specifically have only one click handler  ?

Reply:
I write about it on topic little later.